database-access/src/test/java/org/kingsski/database/dao/jdbc/AbstractDaoIT.java

```diff
@@ -0,0 +1,37 @@
+package org.kingsski.database.dao.jdbc;
+
+import org.junit.runner.RunWith;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import javax.annotation.Resource;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@TestPropertySource("classpath:test.properties")
+@ContextConfiguration(classes = IntegrationTestConfig.class)
+public abstract class AbstractDaoIT {
```

Comment:
Potentially worth a javadoc or at least comments describing what this does and how best to use it

Reply:
Done